.github/workflows/build_and_test.yml

```diff
@@ -1081,13 +1081,27 @@ jobs:
           # Retry script for 5XX server errors: https://github.com/actions/github-script#retries
           retries: 5
           script: |
-            github.rest.git.createRef({
+            await github.rest.git.createRef({
               owner: context.repo.owner,
               repo: context.repo.repo,
               ref: "refs/tags/${{ needs.tag.outputs.build-tag }}",
               sha: context.sha,
             })
 
+      - name: Create GitHub release
+        if: github.ref_name == 'release'
+        uses: actions/github-script@v6
+        with:
+          # Retry script for 5XX server errors: https://github.com/actions/github-script#retries
+          retries: 5
+          script: |
```

Comment:
TODO: We need to check that the default token is enough here; otherwise we need to use  the following:
```suggestion
          github-token: ${{ secrets.CI_ACCESS_TOKEN }}
          script: |
```

Reply:
It works:

<img width="1159" alt="Screenshot 2023-09-09 at 20 29 26" src="https://github.com/neondatabase/neon/assets/864213/a948dfd7-772f-44b0-b05d-44effb2d7552">